packages/dialtone-tokens/build-docs.js

```diff
@@ -18,37 +19,57 @@ export function buildDocs(platformName, theme, currentObj) {
 
   if (tokenValue && tokenPath) {
     const tokenKey = tokenPath.join('/');
+
     docTokens[theme][tokenKey] = {
       ...docTokens[theme][tokenKey],
       [platformName]: {
         name: formatTokenName(platformName, tokenName),
         value: tokenValue,
         description: tokenDescription,
-      }
-    }
+        keywords: getTokenKeywords(tokenPath),
+      },
+    };
     return null;
   }
 
   for (const key in currentObj) {
-    if (!currentObj.hasOwnProperty(key))
-      continue;
+    if (!Object.prototype.hasOwnProperty.call(currentObj, key)) { continue; }
     buildDocs(platformName, theme, currentObj[key]);
   }
 }
 
-function formatTokenName(platformName, tokenName) {
+/**
+ * Gets the keywords from the keywords.json file
+ * for the token category / subcategory
+ * @param {Array} tokenPath The path that has first the category, then the subcategory (optional), then the name
+ * @returns {Array|undefined} Array with the keywords, or undefined if there are none
+ */
+function getTokenKeywords (tokenPath) {
+  const keywords = [];
+  const keywordsCategory = keywordsJson[tokenPath[0]];
+  const keywordsSubcategory = keywordsCategory && tokenPath[1] && keywordsCategory[tokenPath[1]];
+  if (keywordsCategory) {
+    if (keywordsCategory.keywords) keywords.push(...keywordsCategory.keywords);
+  }
+  if (keywordsSubcategory) {
+    if (keywordsSubcategory.keywords) keywords.push(...keywordsSubcategory.keywords);
+  }
```

Comment:
We could probably make this a bit more dynamic using recursion, but if we are always just going to have category and subcategory (doesn't go deeper than that) then it's probably not worth bothering.

Reply:
Good call, just in case I did it that way